packages/Liquid-Tests.package/LQPollTests.class/instance/testIDsAreUUIDs.st

```diff
@@ -0,0 +1,10 @@
+tests
+testIDsAreUUIDs	
```

Comment:
```suggestion
testIDsAreUUIDsSoTheyDontCollide
```
[dust] would make the comment below obsolete

Reply:
I have changed the naming, good suggestion

Reply:
Just to test out how exactly the "suggest changes feature" works - I have marked this comment as resolved, but as you can see below, the suggested change is still blocking the Review/Merge, since I did not apply it from within this comment. However, since the conversation has been marked as resolved, the change cannot be committed anymore (mouseover will say this as a tooltip). Now I see two possibilities:
1: You have the option to change the type of your review from "suggest changes" to "approve" - then my question would be: Why do you need to do this manually?!?
2: We can delete this comment, which is totally counter-intuitive to the "resolve"-feature. If the "skn0tt requested changes" badge below stays after deleting the comment we know that you need to review the PR again, or change the type of the approval.
Dang, this message got long. I think you get that I don't like the feature...

Reply:
Just pinging @Skn0tt in case you miss the above comment since I marked the conversation as resolved